src/helpers/build-image-service-url.js

```diff
@@ -1,12 +1,11 @@
 import qs from 'querystring';
 
-export default (url, params = {}) => {
+export default (url, params = {}, host = 'https://next-geebee.ft.com/image/v1/images/raw/') => {
```

Comment:
can we put this `host` in an object (in case there's additional options in the future)

```
export default (url, params = {}, { host = 'https://next-geebee.ft.com/image/v1/images/raw/' } = { }) => {
```